src/platform/testing/e2e/nightwatch-commands/axeCheck.js

```diff
@@ -1,5 +1,61 @@
 import axeCore from 'axe-core'; // eslint-disable-line no-unused-vars
 
+// Axe v3.5.4 does not accurately determine the foreground color on the
```

Comment:
Trying to understand this better. Is this a limitation of axe-core?

Reply:
As I mentioned in Slack...

The axe-core library has a lot of color-contrast issues - https://github.com/dequelabs/axe-core/issues?q=is%3Aissue+is%3Aopen+color+contrast - it doesn't appear to correctly identify the background & foreground color of some elements.

Here is one example:

```json
{
  "all": [],
  "any": [{
    "data": {
      "bgColor": "#ffffff",
      "contrastRatio": 1.64,
      "expectedContrastRatio": "4.5:1",
      "fgColor": "#f0bdbf",
      "fontSize": "12.0pt (16px)",
      "fontWeight": "normal",
      "messageKey": null
    },
    "id": "color-contrast",
    "impact": "serious",
    "message": "Element has insufficient color contrast of 1.64 (foreground color: #f0bdbf, background color: #ffffff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1",
    "relatedNodes": [{
      "html": "<body class=\"page-healthcare merger\">",
      "target": ["body"]
    }]
  }],
  "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 1.64 (foreground color: #f0bdbf, background color: #ffffff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1",
  "html": "<span class=\"schemaform-required-span\">(*Required)</span>",
  "impact": "serious",
  "none": [],
  "target": ["#root_medicarePartAEffectiveDate-label > .schemaform-required-span"]
}
```

![Screen Shot 2020-06-11 at 11 46 50 AM](https://user-images.githubusercontent.com/136959/85435759-91b6b180-b54d-11ea-8bf0-3e7e8f097535.png)